packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -26,6 +26,23 @@ export class Loan {
   async createLoanScheme (scheme: CreateLoanScheme, utxos: UTXO[] = []): Promise<string> {
     return await this.client.call('createloanscheme', [scheme.minColRatio, scheme.interestRate, scheme.id, utxos], 'number')
   }
+
+  /**
+   * Set a collateral token transaction.
+   *
+   * @param {SetCollateralToken} collateralToken
+   * @param {string} collateralToken.token Symbol or id of collateral token
+   * @param {BigNumber} collateralToken.factor Collateralization factor
+   * @param {string} collateralToken.priceFeedId txid of oracle feeding the price
+   * @param {number} [collateralToken.activateAfterBlock] changes will be active after the block height (Optional)
```

Comment:
```suggestion
   * @param {number} [collateralToken.activateAfterBlock] changes will be active after the block height
```
Redundant (optional) as [] already means that it's an optional parameter

Reply:
wondering what happened to this😉

Reply:
Forgot to remove. Just removed.  Thanks.